sessionspaces/src/resources/config_maps.rs

```diff
@@ -0,0 +1,93 @@
+use k8s_openapi::api::core::v1::ConfigMap;
+use kube::{
+    api::{ObjectMeta, Patch, PatchParams},
+    Api, Client,
+};
+use ldap3::{LdapConnAsync, Scope, SearchEntry};
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    error::Error,
+};
+use tracing::{info, instrument};
+
+const POLICY_CONFIG: &str = "policy-config";
+
+pub async fn ldap_search(namespace: String) -> Result<String, Box<dyn Error>> {
```

Comment:
This should probably be part of the `permissionables` module as it's getting data